src/C_TO_LOGIC.py

```diff
@@ -11317,5 +11296,5 @@ def GET_C_AST_GLOBAL_DECLS(c_file_ast):
     return variable_defs
 
 
-def GET_TYPE_FROM_LIST(py_type, l):
-    return [i for i in l if type(i) == py_type]
+def GET_TYPE_FROM_LIST(py_type, list):
```

Comment:
Looks good - at first I wasnt sure if list was a legal var name since is also a type? but its all good

maybe can move this func to utilities some time